charts/rancher-istio/1.5.900/charts/galley/values.yaml

```diff
@@ -2,7 +2,7 @@
 # galley configuration
 #
 enabled: true
-replicaCount: 1
+replicaCount: 2
```

Comment:
This is really caused by this: https://github.com/istio/istio/issues/12602#issuecomment-489449657
I don't think just bumping the replica count for everyone is the right solution though, even though its a good workaround. Istio is already heavy as it is, this could crash existing clusters. Perhaps we could lower/delete the disruption budget ? On the other side, this version is deprecated and there is a workaround, so we shouldn't put too much time into fixing a problem that the upstream team won't fix.

Reply:
That makes sense to me.  Could we maybe add a configurable for the disruption budget that manages the minimum scale accordingly for the user and lets them know of the increased resource requirements?

Reply:
I think it's worth having a fix for this as it's... disruptive.. (sorry, not sorry)

Reply:
Would a valid fix be to turn `defaultPodDisruptionBudget.enabled` to false ? Changing the PDB `minAvailable` itself is discouraged by istio because people would try to change it on update and its immutable. 
Beyond that, if you mean a single variable to update the scale of each of these pods themselves, the chart doesn't do that right now. We could add it but it feels kinda weird.